packages/router-plugin/src/ast.ts

```diff
@@ -1,4 +1,19 @@
 import * as babel from '@babel/core'
+// @ts-expect-error
+import _babelPluginJsx from '@babel/plugin-syntax-jsx'
```

Comment:
Are the changes here actually needed? What was broken before?

Reply:
Yes, babels plugin resolver was not working here.